helm/hpcc/templates/_helpers.tpl

```diff
@@ -851,7 +824,8 @@ data:
 {{- include "hpcc.generateLoggingConfig" . | indent 6 }}
 {{ include "hpcc.generateVaultConfig" . | indent 6 }}
 {{- if .me.storage }}
-      storagePath: {{ include "hpcc.getVolumeMountPrefix" (dict "root" .root "me" .me.storage "name" (printf "sasha-%s" .me.name) ) }}
+ {{- $sashaStoragePlane := .me.storage.plane | default (include "hpcc.getFirstPlaneForCategory" (dict "root" .root "category" "sasha")) }}
```

Comment:
I think as part of this rationalization process, it would not be better to remove the storageType scope in Sasha (which used to be useful to define ephemeral storage), and just have "plane" or "storagePlane".

Reply:
I think you didn't mean the word "not".  do you want to make the changes as a follow on PR?